vendor/cookbooks/rails/attributes/default.rb

```diff
@@ -1,3 +1,4 @@
+default["apt"]["unattended_upgrades"]["enabled"] = false
```

Comment:
I think we can safely remove this line because it's already the default in the recipes of the `apt` package:

https://github.com/opscode-cookbooks/apt/blob/master/attributes/default.rb#L32